### DISCUSSION TITLE
Question about ReEDS Augur

### DISCUSSION THREAD

**Discussion by narata** [NREL/ReEDS-2.0#76] [Q&A: Analysis] [answered]
Hi, ReEDS Team!

I am very impressed with the great function and precision of ReEDS!

I can‘t understand  line 278 in 'C1_existing_curtailment.py ', which is writen as:  '**net_load_adj_mingen = net_load_adj - curt** (1)'.
As the previous statement in ReEDS_Augur A1 and C1: '**net_load_adj' = 'load_Osprey - vre_BA - region_flows + storage_charge** (2)', 
In my opinion, vre_BA contains curt part and used part. Therefore, '**curt**' has been involved in '**vre_BA**'. 

So what the equation (1) and 'net_load_adj_mingen' mean ?

Many thanks!

**Comment by awfrazier** [comment, marked as answer]
Hello

Apologies for the unclear code.

'net_load_adj_mingen' is intended to be the net load/curtailment profile, with net load as positive and curtailment negative. This happens on lines 279 and 280. Incidentally, lines 279 and 280 are overwriting any calculation that occurs on line 278. It appears that line 278 is just an extraneous line of code and it would be more clear if it were rewritten as: 'net_load_adj_mingen = net_load_adj.copy()' with the actual logic and calculation still occurring as written on lines 279 and 280.

**Reply by narata** [reply]
Thanks for your clear answer!  It helps me a lot!
